websites/U/Unicode Character Table/presence.ts

```diff
@@ -0,0 +1,179 @@
+				presenceData.state = `${
+					document.querySelector<HTMLHeadingElement>("h1").textContent
+				} (${
+					document
+						.querySelector<HTMLSpanElement>(".page-content__info-range")
+						.textContent.match(/([A-Z0-9]{4,6}—[A-Z0-9]{4,6})/)[1]
+				})`;
+			}
+			break;
+		}
+		case "emoji": {
+			presenceData.details = "Browsing list of emojis";
+			if (pathSplit[1]) {
+				presenceData.state =
+					document.querySelector<HTMLHeadingElement>("h1").textContent;
+			}
+			break;
+		}
+		case "holidays": {
+			if (pathSplit[1]) {
+				presenceData.details = "Browsing holiday symbols";
+				presenceData.state =
+					document.querySelector<HTMLHeadingElement>("h1").textContent;
+			} else presenceData.details = "Browsing list of holiday sets";
+			break;
+		}
+		case "html-entities": {
+			presenceData.details = "Browsing list of HTML entities";
+			break;
+		}
+		case "kaomoji": {
+			presenceData.details = "Browsing list of kaomoji";
+			break;
+		}
+		case "search": {
+			presenceData.details = "Searching for characters";
+			presenceData.state = new URLSearchParams(search).get("q");
+			break;
+		}
+		case "sets": {
+			presenceData.details = "Browsing symbol sets";
+			if (pathSplit[1]) {
+				presenceData.state =
+					document.querySelector<HTMLHeadingElement>("h1").textContent;
+			}
+			break;
+		}
+		case "text-art": {
+			presenceData.details = "Browsing text art";
+			break;
+		}
+		case "tools": {
+			switch (pathSplit[1] ?? "") {
+				case "": {
+					presenceData.details = "Browsing list of tools";
+					break;
+				}
+				case "braile": {
+					presenceData.details = "Using the Braille translator";
+					break;
+				}
+				case "decoder": {
+					presenceData.details = "Using the Unicode decoder";
+					break;
+				}
+				case "encoder": {
+					presenceData.details = "Using the HTML encoder";
+					break;
+				}
+				case "flip": {
+					presenceData.details = "Using the flip text tool";
+					break;
+				}
+				case "generator": {
+					presenceData.details = "Using the Unicode table generator";
+					break;
+				}
+				case "gothic": {
+					presenceData.details = "Using the Gothic-style text generator";
+					break;
+				}
+				case "morse": {
+					presenceData.details = "Using the Morse code converter";
+					break;
+				}
+				case "nickname-generator": {
+					presenceData.details = "Using the stylish text generator";
+					break;
+				}
+				case "password-generator": {
+					presenceData.details = "Using the password generator";
+					break;
+				}
+				case "register-converter": {
+					presenceData.details = "Using the case converter";
+					break;
+				}
+				case "strikethrough-text": {
+					presenceData.details = "Using the strikethrough text generator";
+					break;
+				}
+				case "text-to-symbols": {
+					presenceData.details = "Using the text to symbols converter";
+					break;
+				}
+				case "word-counter": {
+					presenceData.details = "Using the word counter";
+					break;
+				}
```

Comment:
Simpler solution:
![image](https://user-images.githubusercontent.com/42322979/192960679-6cc0c506-6a8a-4082-84e6-615a86c49f5a.png)
![image](https://user-images.githubusercontent.com/42322979/192960721-3113b0f5-2d25-4d0d-b448-3f70c7ed66b9.png)
```suggestion
				default: {
					presenceData.details = `Using the ${document.querySelector('[class="tool__heading page-heading"]')?.textContent.trim()} tool`
					break;
				}```